alert-combiner-py/src/agent.py

```diff
@@ -145,8 +144,8 @@ def handle_block(block_event: forta_agent.block_event.BlockEvent) -> list:
             thread.start()
 
         # uncomment for local testing; otherwise the process will exit
-        #while (thread.is_alive()):
-        #    pass
+        while (thread.is_alive()):
+            pass
```

Comment:
curious why this is needed for local testing? Also, do we want to keep this commented?

Reply:
yes, this is needed as for local testing the bot code would exit whereas in prod, we wouldnt want that.